Game.js

```diff
@@ -1,385 +1,871 @@
+
+const healthPrice = 10;
+const healthVolumePrice = 20;
+const maskPrice = 15;
+const maskVolumePrice = 25;
+const antisepticPrice = 5;
+const antisepticVolumePrice = 15;
+
+const maxY = canvas.height / 3;
+const imageX = canvas.width / 6;
+const topRowY = 40;
+const lineX = 165;
+const lineY = 60;
+const rectHeigth = 100;
+const rectLength = 250;
+const lineButtonNumber = 3;
+const columnButtonNumber = 2;
+const paddingY = 60;
+const paddingX = 40;
+const imageSize = 125;
+
+const buttonX = canvas.width / 3;
+const buttonY = canvas.height / 3;
+
+const rects = {};
+const buttons = [];
+const topRow = [];
+const shopImages  = [];
+let lines = [];
+const prices = [];
+const rectsName = [];
+
+const coordinatesOfButtonY = [];
+const coordinatesOfButtonX = [];
+
+let ID;
+
+const keyCods = {
+  enter: 13,
+  right: 39,
+  left: 37,
+}
+
+topRow.push(images.money, images.health, images.antiseptic, images.masks);
+
+buttons.push(
+  images.buyPizza, images.increase, images.buyAntiseptic, images.increase,
+  images.buyMasks, images.increase, images.nextLevel
+);
+
+shopImages.push(images.pizzaImage, images.antImage, images.maskImage);
+
+prices.push(
+  healthPrice, healthVolumePrice, antisepticPrice,
+  antisepticVolumePrice, maskPrice, maskVolumePrice
+);
+
+rectsName.push(
+  'buyPizza', 'increasePizzaVolume', 'buyAntiseptic',
+  'increaseAntisepticVolume', 'buyMasks', 'increaseMasksVolume',
+  'nextLevelButton', 'moneyObj', 'healthObj', 'antisepticObj',
+  'masksObj', 'pizzaImage', 'antImage', 'maskImage'
+);
+
+class Images {
+  constructor(y, image) {
+    this.image = image;
+    this.x = imageX;
+    this.y = y;
+  }
+
+  draw() {
+    ctx.drawImage(
+      this.image, 0, 0,
+      this.image.width, this.image.height,
+      this.x, this.y,
+      this.image.width, this.image.height
+    );
+  }
+}
+
+class TopRow extends Images {
+  constructor(x, image, button1, button2) {
+    super(topRowY, image);
+    this.x = x;
+    this.lineX = this.x + lineX;
+    this.lineY = this.y + lineY;
+    if (button2) {
+      this.addOnes = button1;
+      this.addVolume = button2;
+      this.addOnes.obj = this;
+      this.addVolume.obj = this;
+    } else {
+      this.parametr = button1;
+    }
+  }
+
+  drawLine() {
+    ctx.font = '24px Arial';
+    ctx.fillStyle = 'black';
+    if (this.parametr >= 0) {
+      ctx.fillText(`${this.parametr}`, this.lineX, this.lineY);
+    } else {
+      ctx.fillText(`${this.addOnes.parametr}/${this.addVolume.parametr}`,
+        this.lineX, this.lineY);
+    }
+  }
+}
+
+class Buttons extends Images {
+  constructor(x, y, image, parametr, price) {
+    super(y, image);
+    this.x = x;
+    this.parametr = parametr;
+    this.price = price;
+  }
+
+  increase() {
+    this.parametr++;
+  }
+
+  get canIncrease() {
+    if (this === this.obj.addVolume || (this === this.obj.addOnes &&
+      this.parametr < this.obj.addVolume.parametr)) {
+      return true;
+    }
+    return false;
```

Comment:
Don't return true and false, you can just return result of if-condition (remove if-statement)